src/ts/ssh/sshChannel.ts

```diff
@@ -271,6 +276,61 @@ export class SshChannel implements Disposable {
 		}
 	}
 
+	public async sendStderr(data: Buffer, cancellation?: CancellationToken): Promise<void> {
```

Comment:
How about making it flexible: name it `sendExtendedData()` and make `dataTypeCode` an argument?

Reply:
`sendStderr` may then just call it with `1` data type code.

Reply:
@IlyaBiryukov , I pushed changes (see [diff](https://github.com/microsoft/dev-tunnels-ssh/compare/eb479a142f45dc8c7e6a7ace4da2b687280498ae...bede4e0942645b087614102ad703861485d43120?expand=1)) to address this comment

Reply:
Thanks! The change looks good for me. I'll defer to Jason to sign off, he has more comments.